docs/sources/mimir/references/configuration-parameters/index.md

```diff
@@ -1571,6 +1571,11 @@ query_frontend:
   # ruler.query-frontend.grpc-client-config
   [grpc_client_config: <grpc_client>]
 
+  # (experimental) Format to use when retrieving query results from
+  # query-frontends. Supported values: json, protobuf
+  # CLI flag: -ruler.query-frontend.query-result-response-format
```

Comment:
Can you also mention it in the list of experimental features under `docs/sources/mimir/operators-guide/configure/about-versioning.md`? It's non blocking, you can do it in a follow up PR too.

Reply:
Good catch, I've created https://github.com/grafana/mimir/pull/4429 to do this.